doc/version_history.rst

```diff
@@ -2,6 +2,16 @@
 Version History
 ===============
 
+v0.5.3
```

Comment:
maybe v0.6.0? Seems like pretty substantial changes

Reply:
Done in the commit: [117932d](https://github.com/lsst-ts/ts_m2com/pull/13/commits/117932d6029ac3096ad2001c1ae2f5177e8e0b6f). Thanks!